### DISCUSSION TITLE
Intent to launch play current shortcut

### DISCUSSION THREAD

**Discussion by cromulent403** [PaulWoitaschek/Voice#1934] [Q&A] [answered]
Hi,

Currently running version 7.0.12.

I use Tasker to automatically launch Voice when connecting my Bluetooth headphones.

This worked perfectly in Version 6.4.1. 

But doesn't in 7.0.12. 

There is a shortcut available to play current in Voice that I can add to my home screen.

Tasker is able to send intents and I can use that instead to automatically start playback.

What is the intent sent when the play current shortcut is used?

**Comment by PaulWoitaschek** [comment, marked as answer]
Once this gets rolled out:

https://github.com/PaulWoitaschek/Voice/issues/1913

You can call this receiver:

`voice.playback.receiver.MediaButtonReceiver`

with an intent of the name:

`android.intent.action.MEDIA_BUTTON`

and a key event with the action `KeyEvent.ACTION_UP` and the key code `KEYCODE_MEDIA_PLAY_PAUSE`
